lib/experimental/Information/Headers/BaseHeader/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button } from "@/components/Actions/Button"
+import { IconType } from "@/components/Utilities/Icon"
+import {
+  AvatarVariant,
+  renderAvatar,
+} from "@/experimental/Information/Avatars/utils"
+import { cn } from "@/lib/utils"
+
+interface BaseHeaderProps {
+  title: string
+  avatar?: AvatarVariant
+  description?: string
+  eyebrow?: React.ReactNode
+  footer?: React.ReactNode
+  primaryAction?: {
+    label: string
+    icon?: IconType
+    onClick: () => void
+  }
+  secondaryActions?: Array<{
+    label: string
+    icon?: IconType
+    onClick: () => void
+    variant?: "outline" | "critical"
+  }>
+}
+
+export function BaseHeader({
+  title,
+  avatar,
+  description,
+  eyebrow,
+  footer,
+  primaryAction,
+  secondaryActions,
+}: BaseHeaderProps) {
+  return (
+    <div className="flex flex-col items-start justify-start gap-4 p-8 md:flex-row">
+      <div className="flex grow flex-col items-start justify-start gap-4 md:flex-row md:items-center">
+        {avatar && (
+          <div className="flex items-start">
+            {renderAvatar(avatar, "large")}
```

Comment:
Never understood why `renderAvatar` isn't just a component?

Reply:
Oh, it made sense to me, but yeah, it might be better as a component. I can do that in another PR!

Reply:
Components are just syntax sugar in top of functions :). They feel more natural for this use case.